### ISSUE TITLE
Properly train TensorFlow chatbot on movie scripts 

### ISSUE THREAD

**Issue by michaelsterpka** [michaelsterpka/data-science-projects#1]
3 Layers of 256 units
Currently has learning rate: 0.4555
Most recent checkpoint: 17000

**Comment by michaelsterpka** [comment]
Ckpt 17000
"_UNK _UNK _UNK _UNK"
